### ISSUE TITLE
did anything result from this?

### ISSUE THREAD

**Issue by stuebinm** [herrdragon/busTrackingGps#5]
Hi,

I've been thinking about a similar tracking solution (for a small, community-run train line) and stumbled across this repo since it's linked in https://github.com/CUTR-at-USF/awesome-transit.

Since the Readme still says "beta" and the project appears to be abandoned: did anything result from your tests with it / is a version of this still in use or being developed elsewhere? Just thought I'd ask.

**Comment by qtrandev** [comment]
We stopped working on this 5 years ago. This project was abandoned due to lack of manpower. I moved away from Miami and stopped working on it. We had 2 hardware boxes working and it was deployed on the City of Sweetwater trolley buses, but I had to abandon this project since I couldn't physically be there to manage them.

The hardware and C program sending the data worked so the project is still usable if implemented. At one point, we had it publishing data of the Sweetwater Trolley on our Miami map: http://qtrandev.github.io/LeafletTransit/
See https://github.com/qtrandev/MiamiTrolleyInfo for some links related to transit in Miami.
